components/eventing-controller/controllers/backend/reconciler.go

```diff
@@ -300,6 +277,42 @@ func (r *Reconciler) reconcileBEBBackend(ctx context.Context, bebSecret *v1.Secr
 	return ctrl.Result{}, nil
 }
 
+func (r *Reconciler) syncOauth2ClientIDAndSecret(ctx context.Context, desiredStatus eventingv1alpha1.EventingBackendStatus, backend *eventingv1alpha1.EventingBackend) error {
```

Comment:
Should `desiredStatus` be passed by reference?  Since we are modifying it inside.
```
syncOauth2ClientIDAndSecret(ctx context.Context, desiredStatus *eventingv1alpha1.EventingBackendStatus, ...
```

Reply:
Yes that's right! Will change it